pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -265,15 +280,24 @@ func (r *ObjectBucketClaimReconciler) handleDeleteClaim(key client.ObjectKey) er
 		return nil
 	}
 	// see if obc delete is for a newly provisioned bkt vs an existing bkt
-	isDynamicallyProvisioned := isNewBucketByOB(r.internalClient, ob)
 
 	// Call the provisioner's `Revoke` method for old (brownfield) buckets regardless of reclaimPolicy.
 	// Also call `Revoke` for new buckets with a reclaimPolicy other than "Delete".
 	if ob.Spec.ReclaimPolicy == nil {
 		log.Error(nil, "got null reclaimPolicy", "ob", ob.Name)
 		return nil
 	}
+
+	ob, err = updateObjectBucketPhase(r.internalClient, ob, v1alpha1.ObjectBucketClaimStatusPhaseReleased, r.retryInterval, r.retryTimeout)
```

Comment:
no `err` check.

Reply:
should there be a parallel _Released_ phase set for the OBC too, before its deleted?
_Released_ should be a very short-lived Phase. What is the benefit of setting this here and then (after `Delete` or `Revole`) deleting the ob?

Reply:
The OBC can't be updated because it's been deleted.  When we get the event, the API object is already gone.

Marking the OB as `released` gives us some indication an obc deletion occurred but has not completed.